.eslintrc

```diff
@@ -16,7 +16,12 @@
   "extends": ["airbnb", "prettier"],
   "plugins": ["prettier"],
   "rules": {
-    "prettier/prettier": ["error"],
```

Comment:
Can you explain this change?

Reply:
![grafik](https://user-images.githubusercontent.com/88881326/214936985-3abadeb1-0bad-4d46-b095-bddc732868af.png)
without it I get every line marked because the line separator is set to CLRF.

Reply:
fixed